### DISCUSSION TITLE
Hi Everyone .. Got a question .. Have like 150 YouTube links in opera.

### DISCUSSION THREAD

**Discussion by Warlock-Weary** [FreeTubeApp/FreeTube#4243] [General]
Hi Everyone .. Got a question .. Have like 150 YouTube links in opera.
Would be super cool if you could have a import ? or a way to use them in Freetube ..

Or am I missing a way to do this ?

![image](https://github.com/FreeTubeApp/FreeTube/assets/88459215/71cb90f3-15c6-4d84-8fb7-1be190392586)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @Warlock-Weary, importing bookmarks is not something we support at the moment. There is an open issue for this https://github.com/FreeTubeApp/FreeTube/issues/1426 on our issue tracker.
